Example/Shared/Signer/Signer.swift

```diff
@@ -147,11 +147,11 @@ final class Signer {
 
             let userOpHash = try await client.sendTransaction(transaction)
 
-            Task {
-                let userOpReceipt = try await SmartAccount.instance.getClient().waitForUserOperationReceipt(userOperationHash: userOpHash)
-                guard let userOpReceiptSting = userOpReceipt.jsonString else { return }
-                AlertPresenter.present(message: userOpReceiptSting, type: .info)
-            }
+//            Task {
```

Comment:
Don't we still want this?

Reply:
i was testing with releases without that feature, reenabled